staging/src/k8s.io/apiserver/pkg/endpoints/filters/metrics.go

```diff
@@ -66,7 +66,7 @@ var (
 			Buckets:        metrics.ExponentialBuckets(0.001, 2, 15),
 			StabilityLevel: metrics.ALPHA,
 		},
-		[]string{"result"},
+		[]string{"result", "authenticator"},
```

Comment:
what's the cardinality of `authenticator`?

Reply:
It has 8 possible values, see staging/src/k8s.io/apiserver/pkg/authentication/authenticator/interfaces.go.